.github/workflows/scripts/build_linux_artifacts.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+set -e
+
+REPODIR=$(git rev-parse --show-toplevel)
+
+TAG=${TAG:-$(git log -n1 --pretty=%h)}
+OUTDIR=${OUTDIR:-"${REPODIR}/artifacts"}
+
+TARCMD=tar
+if [[ $(uname -s) == "Darwin" ]]; then
```

Comment:
This was mostly for test convenience. As a developer it was convenient to build the linux artifacts from my box instead of spinning up yet-another-container. I'm ok removing it 🤷

Reply:
I'm inclined to keep it as is to make the developer experience easier.

Reply:
In that case, it makes sense to keep, but since it's just for dev purposes, I would suggest adding a comment to clarify that.